cli/util/errors.go

```diff
@@ -5,4 +5,8 @@ import "errors"
 var (
 	// ErrCmdAbort is reported when user aborts the program.
 	ErrCmdAbort = errors.New("aborted by user")
+
+	// ErrNoTtInstalled is reported when program cannot find tt binary
+	// file with name provided.
+	ErrNoTtInstalled = errors.New("there is no tt installed")
```

Comment:
This doesn't look like some kind of general-public error so it's better define it on the `uninstall` module level like a private error.

Reply:
Have deleted it and put in uninstall.go as private error.